detection/notebooks/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>Might be worth making a quick note (like one or two sentences) about autoscaling cluster and its cost impact since we're spinning up V100s.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Added